### PR TITLE
Added preview accepted status

### PR DESCRIPTION
**Pull request by asadazam93**
### Description
The course run detail page widget sidebar was missing preview accepted status for publisher. I have added a condition to fix it. However, this is not the proper solution as it makes the code unclean and unnecessarily complicated. I have added complete details of the proper solution but since that is a complicated task that is to be done later. 

**Sandbox**
N/A

### Reviewers
If you've been tagged for review, please check your corresponding box once you've given the :+1:.
- [ ] @attiyaIshaque 
- [ ] @Rabia23 

### Post-review
- [ ] Rebase and squash commits

### CODE REVIEW

File: course_discovery/apps/publisher/templates/publisher/_approval_widget.html [hunk 1]
```diff
@@ -41,6 +41,8 @@ <h5 class="hd-5 emphasized approvals-heading">{% trans "REVIEWS" %}</h5>
                         {% elif course_run.course.publisher == request.user %}
                             {% if course_run.course_run_state.owner_role == 'course_team' %}
                                 <span class="preview-status">{% trans "Submitted for review" %}</span>
+                            {% elif course_run.course_run_state.owner_role == 'publisher' and course_run.course_run_state.preview_accepted %}
```
Comment (awaisdar001) [inline course_discovery/apps/publisher/templates/publisher/_approval_widget.html]: Can we have `course_run.course_run_state.owner_role` in a varible and use it?

File: course_discovery/apps/publisher/templates/publisher/_approval_widget.html [hunk 2]
```diff
@@ -41,6 +41,8 @@ <h5 class="hd-5 emphasized approvals-heading">{% trans "REVIEWS" %}</h5>
                         {% elif course_run.course.publisher == request.user %}
```
Comment (awaisdar001) [inline course_discovery/apps/publisher/templates/publisher/_approval_widget.html:41]: also `course_run.course.publisher == request.user` in a variable?

File: course_discovery/apps/publisher/templates/publisher/_approval_widget.html (hunk 1 above)
Comment (awaisdar001) [inline course_discovery/apps/publisher/templates/publisher/_approval_widget.html]: What about this? and moving all the logic in `course_run` model's class method `preview_status`? 
`<span class="preview-status">{{course_run.preview_status}}</span>`

It looks cleaner.

**Review by awaisdar001 — CHANGES_REQUESTED** [review]

**Review by awaisdar001 — APPROVED** [review]

**Comment by awaisdar001** [comment]
Please add tests as well.

**Review by attiyaIshaque — APPROVED** [review]
Please squash your commits and update your branch with the latest code. 
Once the tests passed. 👍

**Pull request merged**